src/lang-promql/complete/hybrid.ts

```diff
@@ -427,6 +473,7 @@ export class HybridComplete implements CompleteStrategy {
           });
           break;
         case ContextKind.Duration:
+          span = false;
```

Comment:
Maybe not super important right now, but we might also want a custom span regex for e.g. label values, which can be arbitrary strings?

Reply:
That's possible I'm just not sure about the value here

Reply:
Yeah I would also have to look deeper, maybe it would just be `.*`? Anyway, not important for this PR.

Reply:
mmm ok, then I will let you take a look once you have the time. I think I missed something here.